servicetalk-loadbalancer-experimental/src/main/java/io/servicetalk/loadbalancer/LoadBalancerPolicies.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright © 2023 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.loadbalancer;
+
+/**
+ * A collections of factories for constructing a {@link LoadBalancingPolicy}.
+ */
+public final class LoadBalancerPolicies {
+
+    private LoadBalancerPolicies() {
+        // no instances.
+    }
+
+    /**
+     * Builder for the round-robin {@link LoadBalancingPolicy}.
+     * Round-robin load balancing is a strategy that maximizes fairness of the request distribution. This comes at the
+     * cost of being unable to bias toward better performing hosts and can only leverage the course grained
+     * healthy/unhealthy status of a host.
+     * @return a builder for the round-robin {@link LoadBalancingPolicy}.
+     */
+    public static RoundRobinLoadBalancingPolicy.Builder roundRobin() {
```

Comment:
If we are adding `LoadBalancerPolicies`, we should follow the pattern of `CapacityLimiters` by making the builders a top-level public final classes that build an instance of `LoadBalancingPolicy` interface (the actual types will be an internal detail).

Reply:
To be safe, we can go through deprecation in experimental module for 1 release

Reply:
Done.